core/client/ui/settings-character.js

```diff
@@ -3,8 +3,9 @@ Template.settingsCharacter.onCreated(() => {
 });
 
 Template.settingsCharacter.helpers({
-  getAllImages() {
-    return Characters.find({ category: Session.get('settings-character-category'), $or: [{ hide: { $exists: false } }, { hide: false }] }).fetch();
+  getAllImages(category) {
+    if (category === undefined) category = Session.get('settings-character-category');
+    return Characters.find({ category: category, $or: [{ hide: { $exists: false } }, { hide: false }] }).fetch();
```

Comment:
Wouldn't a `count` be better in terms of performance? I don't know if front-end mini-mongo makes a difference

Reply:
Yeah, I ended up to the conclusion that it probably doesn't make a difference and tried to make the diff as small as possible instead, but I can add a `countImages` helper instead if you prefer (`getAllImages` is used in other places where the actual documents are used).

Reply:
I don't think it's necessary, it won't change much and if it works it's cool🙂